<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>641</width>
     <height>521</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>639</width>
      <height>519</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pay_btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Thanh Toán</string>
   </property>
  </widget>
  <widget class="QLabel" name="total">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>560</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tổng</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
